<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ITrackFilterWidget</class>
 <widget class="QWidget" name="ITrackFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listFilters">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushResetFilterList">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset Filter List</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/iconClear16x16.png</normaloff>:/icons/iconClear16x16.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushApply">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply Filters</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/iconWizzard16x16.png</normaloff>:/icons/iconWizzard16x16.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/iconTrack16x16.png</normaloff>:/icons/iconTrack16x16.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Points</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>727</width>
            <height>307</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupReducePoints1">
             <property name="title">
              <string>Reduce Points</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Hide track points if:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Distance to previous point less than</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QSpinBox" name="spinDistance1">
                  <property name="suffix">
                   <string notr="true">m</string>
                  </property>
                  <property name="maximum">
                   <number>999</number>
                  </property>
                  <property name="value">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboMeterFeet1">
                  <item>
                   <property name="text">
                    <string notr="true">m</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">ft</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Azimuth from previous point less than</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QSpinBox" name="spinAzimuthDelta1">
                <property name="suffix">
                 <string>Â°</string>
                </property>
                <property name="maximum">
                 <number>90</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QToolButton" name="toolAddHidePoints1">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/iconAdd16x16.png</normaloff>:/icons/iconAdd16x16.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupSmoothProfile1">
             <property name="title">
              <string>Smooth Profile</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QToolButton" name="toolAddSmoothProfile1">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/icons/iconAdd16x16.png</normaloff>:/icons/iconAdd16x16.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Change elevation data of track:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Median filter over</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinSmoothProfileTabs1">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
                <property name="singleStep">
                 <number>2</number>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>points</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/editcut.png</normaloff>:/icons/editcut.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Split</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/cache/Virtual-Cache.png</normaloff>:/icons/cache/Virtual-Cache.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Annonymous</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
